ang/crmFieldMetadata/crmRenderCheckbox.html

```diff
@@ -1,7 +1,22 @@
+<!-- TODO: Is this necessary? Under what circumstances would there be no options? -->
 <input ng-if="!field.options.length" type="checkbox" ng-model="$parent.model"  name="{{field.name}}" id="{{prefix}}{{field.name}}" ng-required="field.required" />
 
+<!-- Because we are using multiple HTML elements (the checkboxes) to represent
```

Comment:
I guess that's better. Still pretty verbose for a code comment. Might be better to have an issue ID with more information. I still don't understand the whole thing. It said before that the validation required that a model be set. Now it looks like that is implied because validation looks at the model. So, now I'm confused because we are disabling binding, so how can validation look at the model? I thought the model was a dummy because we were disabling binding and because it was generically named "model".  Does "update" imply a direction?